CaloFilters/src/CaloLikelihood_module.cc

```diff
@@ -428,13 +429,15 @@ namespace mu2e {
         // associate to the caloCluster which triggers.  Note there may be other caloClusters which also pass the filter
         // but filtering is by event!
         size_t index = std::distance(caloClusters->begin(), icl);
-        triginfo->_caloCluster = art::Ptr<CaloCluster>(clH, index);
-        
+	if (trig_ind < TriggerInfo::MaxNObj){        
+	  triginfo->_caloClusters[trig_ind] = art::Ptr<CaloCluster>(clH, index);
```

Comment:
Is this left over from the array implementation?  It seems to be missing push_back.

Reply:
Hi Dave, I cross-checked it. I updated it in the last commit. I'm not sure why gitHub is not updating this window. here you can see that the code looks fine: https://github.com/Mu2e/Offline/pull/194/commits/e75479ca4eed98df50e004fadbb9708a3a8cb24e